src/platforms/common/enriching-events/identify-user.mdx

```diff
@@ -35,13 +35,21 @@ Users consist of a few critical pieces of information that construct a unique id
 
 `ip_address`
 
-: The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user. Sentry will attempt to pull this from the HTTP request data, if available. That might require <PlatformIdentifier name="send-default-pii" /> set to `true` in the SDK options. <PlatformSection notSupported={["php"]}> If set to `"{{auto}}"`, Sentry will infer the IP address from the connection between your app and Sentry's server, which is useful for client-side applications such as JavaScript applications running on Web browsers, desktop applications, or mobile apps. </PlatformSection>
+: The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user. Sentry will attempt to pull this from the HTTP request data (`request.env.REMOTE_ADDR` field in JSON), if available. That might require <PlatformIdentifier name="send-default-pii" /> set to `true` in the SDK options.
 
-<PlatformSection supported={["apple"]}>
+If the user's `ip_address` is set to `"{{auto}}"`, Sentry will infer the IP address from the connection between your app and Sentry's server. For certain SDKs, particularly iOS and frontend JavaScript, this is the default behavior and requires no further configuration. For most other SDKs the default is not to collect any IP address. If you find that you're missing the user's IP address, setting this value to `"{{auto}}"` explicitly might help.
 
-Events from the SDK for Apple already consider the incoming connection as the user's IP address. To opt-out from this, you can use Sentry's [server-side data](/product/data-management-settings/scrubbing/) scrubbing to remove `$user.ip_address`.
+<!--
+SDK devs who read this and wonder why their SDK does not behave like the JS
+SDK: Relay changes the default value from `null` to `{{auto}}` for the
+platforms javascript, cocoa, objc. If you want to collect IP addresses by
+default because you're building a frontend/desktop SDK, just send {{auto}} by
+default. We don't really want to special-case more platforms.
```

Comment:
nit: are developers usually referencing sentry docs in a way that would make comments like these visible to them? using the native documents as an example (https://sentry-docs-git-doc-ip-auto.sentry.dev/platforms/native/enriching-events/identify-user/) it's not immediately obvious to me how i would find this bit of information without directly referencing the source document. this looks like useful information, but it's squirreled away in a place that doesn't appear to be too discoverable for those that it would help.

Reply:
i now went with less technical phrasing such that it can be shown in user docs